### PR TITLE
can configure value grid string via commands

### PR DESCRIPTION
**Pull request by colombod**
send localization via api call

### CODE REVIEW

**Comment by ocallesp** [comment]
What is the difference between this PR and [the previous one](https://github.com/dotnet/interactive/pull/2812) ?

File: src/polyglot-notebooks-ui-components/src/components/VariableGrid.tsx [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
 import React from "react";
-import { VariableGridRow } from "../contracts/types";
+import { GridLocalization, VariableGridRow } from "../contracts/types";
 
 
 export type VariableGridProps = {
     rows: VariableGridRow[]
 };
 
+
```
Comment (ocallesp) [inline src/polyglot-notebooks-ui-components/src/components/VariableGrid.tsx:12]: nit: extra blank lines

**Review by ocallesp — APPROVED** [review]

**Comment by colombod** [comment]
> What is the difference between this PR and [the previous one](https://github.com/dotnet/interactive/pull/2812) ?

None, the previous one was closed because I deleted the branch by mistake

**Comment by colombod** [comment]
@jonsequitur i ma proposing to merge this in or merging / rebasing into @ocallesp open pr. what do you think?

**Comment by ocallesp** [comment]
> @jonsequitur i ma proposing to merge this in or merging / rebasing into @ocallesp open pr. what do you think?

sure.
Moved to https://github.com/dotnet/interactive/pull/2800/commits/48cdbfdc5007f4ff79242b033f08be0ea08fb9f4

**Pull request closed without merging**